multiaccounts/accounts/database.go

```diff
@@ -35,6 +36,47 @@ type Account struct {
 	Color     string         `json:"color"`
 }
 
+type Settings struct {
+	// required
+	Address                types.Address    `json:"address"`
+	ChaosMode              bool             `json:"chaos-mode?,omitempty"`
+	Currency               string           `json:"currency,omitempty"`
+	CurrentNetwork         string           `json:"networks/current-network"`
+	CustomBootnodes        *json.RawMessage `json:"custom-boot-nodes,omitempty"`
```

Comment:
`json.RawMessage` is alias of `[]byte` so it's already a pointer because slices are pointers. `json.RawMessage` should be enough.

Reply:
it's because it's not just json.RawMessage it can also be nil, when calling getSettings if it's not a pointer you get:
```
        	Error:      	Received unexpected error:
        	            	sql: Scan error on column index 4, name "custom_bootnodes": unsupported Scan, storing driver.Value type <nil> into type *json.RawMessage
```